app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        render json: notes_paged, status: :ok, each_serializer: IndexNoteSerializer
+      end
+
+      def show
+        render json: show_note, status: :ok, serializer: NoteSerializer
+      end
+
+      private
+
+      def show_note
```

Comment:
```suggestion
      def note
```
Lo que devuelve el método es un note directamente. Mantengámos los nombres lo más cercano y descriptivo posible 🚀 .
Por otro lado, te diría que este método está de más. Son muy pocos caracteres, no va a hacer que rubocop chille por el método Show y es super claro lo que hace.